src/python/foglamp/core/api.py

```diff
@@ -6,25 +6,21 @@
 
 import time
 from aiohttp import web
-from foglamp import server
 
 __author__ = "Amarendra K. Sinha"
 __copyright__ = "Copyright (c) 2017 OSIsoft, LLC"
 __license__ = "Apache 2.0"
 __version__ = "${VERSION}"
 
+__start_time = time.time()
+
+
 async def ping(request):
     """
     :return: basic health information json payload
     {'uptime': 32892} Time in seconds since FogLAMP started
     """
 
-    # Since foglamp can be started in foreground or as a daemon,
-    # need to check for both foglampd and foglamp
-    process_info = server.find_process_info('foglampd') or server.find_process_info('foglamp')
-
-    since_started = 0
-    if process_info is not None:
-        since_started = time.time() - process_info['start_time']
+    since_started = time.time() - __start_time
```

Comment:
We currently haven't decided on what the process name would be. I'm not sure the entry point stuff will be used. Depends on how the snap will work. The process that's running the core REST server is the code that has __start_time, so there is no need to look for a process. That would be needed only if core_daemon had a function to report the uptime which thankfully it doesn't.